plugins/version.txt

```diff
@@ -1 +1 @@
-14
\ No newline at end of file
+14
```

Comment:
@arilivigni Can you please increment this. I think this will be required to load the plugin again and then we will et the new set.

Reply:
@piyush-garg done but that shouldn't affect testing the image.